libraries/AP_Mount/AP_Mount_Alexmos.cpp

```diff
@@ -249,7 +262,12 @@ void AP_Mount_Alexmos::parse_body()
             if (_state._yaw_input_mode != AP_Mount::Input_Mode_Angle_Absolute_Frame) {
                 _current_angle.z = VALUE_TO_DEGREE(_buffer.angles_ext.stator_rotor_angle_yaw);
             }
-            break;
+
+            // make yaw encoder value visible outside the AP_Mount class
+            AP_Mount *mount = AP::mount();
+            mount->yaw_encoder_readback = _current_angle.z;
```

Comment:
Check pointer (even though I think it should be fine in this class)

Reply:
Done